django/template/defaulttags.py

```diff
@@ -578,10 +615,14 @@ def __repr__(self):
         return "<WithNode>"
 
     def render(self, context):
+        return ''.join(self.stream(context))
```

Comment:
I guess if we change that, the stream API won't work by default for tags that don't specify a ``stream`` method.